app/managers.py

```diff
@@ -0,0 +1,33 @@
+import sqlite3
+
+from models import Actor
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.Connection("cinema.sqlite")
```

Comment:
The `sqlite3.Connection` should be `sqlite3.connect` to properly establish a connection to the database. This is a critical issue that will prevent the database operations from working correctly.